### PR TITLE
Increase Code coverage

### PR DESCRIPTION
**Pull request by arramesh42**
Added unit tests to increase code coverage
In cases where the error/branch was not directly reachable, used a mock variable to simulate the error.
Before adding these tests, 
before

After adding these tests.
after

### CODE REVIEW

**Comment by arramesh42** [comment]
#17

File: src/famfs_lib.c [hunk 1]
```diff
@@ -3209,13 +3209,14 @@ famfs_cp(struct famfs_locked_log *lp,
 			char destpath[PATH_MAX];
 			char src[PATH_MAX];
 
-			if (verbose > 1)
+			if (verbose)
```
Comment (jagalactic) [inline src/famfs_lib.c:3212]: Why did you drop (verbose > 1)? The intent is that this only gets printed if verbose is > 1. If that branch was not covered, it is easy to cover by adding a "famfs cp -vv" (which I suspect we already have)

File: test/famfs_unit.c [hunk 2]
```diff
@@ -97,7 +97,7 @@ int create_mock_famfs_instance(
 	memset(logp, 0, FAMFS_LOG_LEN);
 
 	/* First mkfs should succeed */
-	rc = __famfs_mkfs("/dev/dax0.0", sb, logp, device_size, 0, 0);
+	rc = __famfs_mkfs("/tmp/famfs", sb, logp, device_size, 0, 0);
```
Comment (jagalactic) [inline test/famfs_unit.c:100]: why this change? First argument of __famfs_mkfs is the dax device

Comment (arramesh42) [reply test/famfs_unit.c:100]: I thought since we created a mock famfs, that should be used to do mkfs.
And also I thouhgt that /dev/dax device may not be available for the unit tests. (that it is available for smokes)

Will change it back...

File: src/famfs_lib.c (hunk 1 above)
Comment (arramesh42) [reply src/famfs_lib.c:3212]: Ok, I had passed the verbose=1 option from the test and it did not cover the path...  Also I thought passing 1 should enable it as I don't see options to be verbose at different log levels (like 1, 2, 3... n), in code as of now anything greater than 1 is essentially same..

Will change it back...

**Comment by arramesh42** [comment]
Addressed the review comments.
